core/modules/user/views/views_plugin_row_user_view.inc

```diff
@@ -34,7 +34,7 @@ class views_plugin_row_user_view extends views_plugin_row {
       '#title' => t('View mode'),
       '#default_value' => $this->options['view_mode'],
      );
-    $form['help']['#markup'] = t("Display the user with standard user view. It might be necessary to add a user-profile.tpl.php in your themes template folder, because the default <a href=\"@user-profile-api-link\">user-profile</a>e template don't show the username per default.", array('@user-profile-api-link' => url('http://api.drupal.org/api/drupal/modules--user--user-profile.tpl.php/7')));
+    $form['help']['#markup'] = t("Display the user account using the standard user account view. It might be necessary to add a user-profile.tpl.php file in your themes template folder, because the default <a href=\"@user-profile-api-link\">user-profile</a> template doesn't show the username per default.", array('@user-profile-api-link' => url('https://api.backdropcms.org/api/backdrop/core%21modules%21user%21templates%21user-profile.tpl.php/1')));
```

Comment:
Can we please remove ` per default`, since we said `the default [...] template` before, it's redundant. Either that, or we could remove the first "default" and change this one to "by default" instead.